docs/src/pages/WebRatingPage/WebRatingPage.js

```diff
@@ -171,6 +171,32 @@ const components = [
       </div>,
     ],
   },
+  {
+    id: 'title',
+    title: 'Title only ratings',
+    blurb: [
+      <Paragraph>
+        The rating component&apos;s can support showing only title.
```

Comment:
```suggestion
        The rating component can support showing only title.
```

Not an appropriate use of an apostrophe 😉